src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderConfig.tsx

```diff
@@ -276,74 +276,82 @@ export const LimitOrderConfig = ({
     return bnOrZero(sellAmountCryptoPrecision).div(limitPrice.sellAssetDenomination).toString()
   }, [limitPrice, priceDirection, sellAmountCryptoPrecision])
 
-  const limitOrderExplanation = useMemo(() => {
-    if (
-      isLoading ||
-      bnOrZero(sellAmountCryptoPrecision).isZero() ||
-      bnOrZero(limitPriceForSelectedPriceDirection).isZero()
-    )
-      return null
-
-    const assetSymbol =
-      priceDirection === PriceDirection.BuyAssetDenomination ? sellAsset.symbol : buyAsset.symbol
+  const assetSymbol = useMemo(
+    () =>
+      priceDirection === PriceDirection.BuyAssetDenomination ? sellAsset.symbol : buyAsset.symbol,
+    [priceDirection, sellAsset.symbol, buyAsset.symbol],
+  )
 
-    return (
-      <CText fontSize='sm' lineHeight='1.6'>
-        <Text
-          as='span'
-          color='text.subtle'
-          translation='limitOrder.explanation.priceReaches.prefix'
-        />
+  const humanReadableExplanationComponents = useMemo(
+    () => ({
+      assetSymbol: (
         <CText as='span' color='white' fontWeight='medium'>
-          {' '}
           {assetSymbol}
         </CText>
-        <Text
-          as='span'
-          color='text.subtle'
-          translation='limitOrder.explanation.priceReaches.middle'
-        />
+      ),
+      limitPrice: (
         <Amount.Crypto
           as='span'
           color='white'
-          ml={1}
           fontWeight='medium'
           value={limitPriceForSelectedPriceDirection}
           symbol={priceAsset.symbol}
           omitDecimalTrailingZeros
         />
-        <Text as='span' color='text.subtle' translation='limitOrder.explanation.orderWill.prefix' />
+      ),
+      sellAmount: (
         <Amount.Crypto
           as='span'
           color='white'
-          ml={1}
           fontWeight='medium'
           value={sellAmountCryptoPrecision}
           symbol={sellAsset.symbol}
           omitDecimalTrailingZeros
         />
-        <Text as='span' color='text.subtle' translation='limitOrder.explanation.ensuring.prefix' />
+      ),
+      receiveAmount: (
         <Amount.Crypto
           as='span'
           color='white'
-          ml={1}
           fontWeight='medium'
           value={receiveAmountFormatted}
           symbol={buyAsset.symbol}
           omitDecimalTrailingZeros
         />
-        <Text as='span' color='text.subtle' translation='limitOrder.explanation.ensuring.suffix' />
-      </CText>
+      ),
+    }),
+    [
+      assetSymbol,
+      limitPriceForSelectedPriceDirection,
+      priceAsset.symbol,
+      sellAmountCryptoPrecision,
+      sellAsset.symbol,
+      receiveAmountFormatted,
+      buyAsset.symbol,
+    ],
+  )
+  const limitOrderExplanation = useMemo(() => {
```

Comment:
ocd: I know this is just moved around but `explainer` terminology for OCD purposes?

Reply:
Done in https://github.com/shapeshift/web/pull/9075/commits/f7c9f4de0b138f8dd5005e524de395376f738b22